pylint/testutils/lint_module_test.py

```diff
@@ -180,15 +185,18 @@ def _get_actual(self) -> Tuple["MessageCounter", List[OutputLine]]:
                 msg.symbol != "fatal"
             ), f"Pylint analysis failed because of '{msg.msg}'"
             received_msgs[msg.line, msg.symbol] += 1
-            received_output_lines.append(OutputLine.from_msg(msg))
+            received_output_lines.append(OutputLine.from_msg(msg, check_end_position))
         return received_msgs, received_output_lines
 
```

Comment:
```suggestion

    def should_check_end_position(self):
        """Should we check the end position ?"""
        return sys.version_info >= self._test_file.options["min_pyver_end_position"]

```

Could we create a function to avoid changing the signature and passing the ``check_end_position`` around ?

Reply:
I thought that doing this once when loading the file was better than doing it for each individual `OutputLine`, that's why I pass it around.

However, I will add a default value so this doesn't become a breaking change.

Reply:
Maybe we can create an attribute in the constructor ? cached property is available only from 3.8 on https://docs.python.org/3.8/library/functools.html?highlight=#functools.cached_property

Reply:
https://github.com/PyCQA/pylint/pull/5386/commits/602b8ce9abc22c8fb7b88e4cb57be358d5b73a24